mvpa/cosmo_crossvalidate.m

```diff
@@ -16,6 +16,14 @@
 %                       first or second dimension of ds. Normalization
 %                       parameters are estimated using the training data
 %                       and applied to the testing data.
+%     .pca              optional, (default: false) Set to true
```

Comment:
Is there a use case for using pca but not using pca_explained_count or pca_explained_ratio? How about removing the .pca option, which is implied whenever pca_explained_count or pca_explained_ratio is provided?